### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by dolanmiu**

### CODE REVIEW

**Comment by reda-alaoui** [comment]
Travis produced the following error:
```
Error: Cannot find module '/home/travis/build/Cosium/dry-dry/dist/index.js'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Function.Module.runMain (module.js:701:10)
    at startup (bootstrap_node.js:190:16)
    at bootstrap_node.js:662:3
```

It seems that we need to user `npm build` instead of `npm install` in the install step, otherwise the dist is not generated.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "stable"
+install:
+  - npm install
```
Comment (reda-alaoui) [inline .travis.yml]: Can you try with `npm run build` to see if the build pass?

Comment (dolanmiu) [reply .travis.yml]: done

**Comment by reda-alaoui** [comment]
I meant add it not replace it ;)
Thank you !

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+script:
+  - npm run build
```
Comment (reda-alaoui) [inline .travis.yml]: Now it seems that test are not runned.
Can you add `npm test`

Comment (dolanmiu) [reply .travis.yml]: Added :)

**Comment by reda-alaoui** [comment]
https://github.com/Cosium/dry-dry/blob/master/src/index.test.ts#L47 is failing.
The timeout must be increased (i.e. to 30000ms) to make the test pass.

**Comment by reda-alaoui** [comment]
Well done.
Thank you :+1:

**Pull request merged**